radar_control/radar_control.py

```diff
@@ -411,6 +412,20 @@ def radar():
                             if first_time:
                                 for pulse_index, pulse_dict in \
                                         enumerate(sequence_dict_list[sequence_index]):
+
+                                    if __debug__:
+                                        if nave == 0 and pulse_index == 0:
```

Comment:
This is assuming that all pulses are repeats and there is only one slice in the integration period. If we are running normalscan this will be the case. I can build in checks for this.